### ISSUE TITLE
Landscape mode

### ISSUE THREAD

**Issue by florianwittmann** [VeggieSailor/VeggieSailor#23]
As a user I want to use my phone in landscape mode.

**Comment by bluszcz** [comment]
Good idea, should be made together with changing layout to be more flexible for tablets also.

**Comment by florianwittmann** [comment]
I guess tablet optimizing is another thing, as there is no sdk for tablet yet. But yes, that's important in near future, too.

**Comment by bluszcz** [comment]
Ok, I am cool with https://github.com/VeggieSailor/VeggieSailor/issues/24
